modules/clusterpool/Makefile

```diff
@@ -5,6 +5,11 @@ CLUSTERPOOL_HOST_API ?=
 CLUSTERPOOL_HOST_USER ?=
 CLUSTERPOOL_HOST_PASS ?=
 CLUSTERPOOL_HOST_TOKEN ?=
+ifdef CLUSTERPOOL_HOST_KUBE_SECRET
+  CLUSTERPOOL_HOST_USER := ""
+  CLUSTERPOOL_HOST_PASS := ""
+  CLUSTERPOOL_HOST_TOKEN := $(shell cat /etc/$(CLUSTERPOOL_HOST_KUBE_SECRET)/token)
```

Comment:
I understand the logic here, but what if you're not running inside prow, but you still have a `CLUSTERPOOL_HOST_KUBE_SECRET` defined?  It doesn't seem likely it would live in `/etc` in that case.  Does prow give some other indication that you're in its midst - so we could check for prow-ness and _then_ look in `/etc`?  Or, instead look for the definition of `CLUSTERPOOL_HOST_KUBE_SECRET_PATH` that defines the path to `token`?

Reply:
The way to check you're running in Prow is to check that the variable CI is set.
This variable should only be set in Prow. Probably should have named it CLUSTERPOOL_HOST_PROW_SECRET, but all the other entries in the step registry are already using _KUBE_SECRET.

Reply:
I'd be happier with `CLUSTERPOOL_HOST_PROW_KUBE_SECRET` in that case - I'm leery of the namespace adjacency of prow-only use of `KUBE` and `CLUSTERPOOL` without it.